test/test_prototype_transforms_consistency.py

```diff
@@ -68,7 +96,18 @@ def parametrization(self):
         [
             ArgsKwargs(32),
             ArgsKwargs((32, 29)),
-            ArgsKwargs((31, 28), interpolation=prototype_transforms.InterpolationMode.BICUBIC),
+            ArgsKwargs((31, 28), interpolation=prototype_transforms.InterpolationMode.NEAREST),
+            ArgsKwargs((33, 26), interpolation=prototype_transforms.InterpolationMode.BICUBIC),
+            # FIXME: these are currently failing, since the new transform only supports the enum. The int input is
```

Comment:
Open question. We can also postpone a decision and potentially fix later.

Reply:
I think we should have dropped int values years ago. Let's keep your comment here as a reminder.